test/e2e/suite/command/resolve_host.go

```diff
@@ -48,7 +48,7 @@ func ResolveFlags(reg string, host string, flagType resolveType) []string {
 	resolveFlag := "resolve"
 	usernameFlag := "username"
 	passwordFlag := "password"
-	plainHttpFlag := "plain-http"
+	plainHttpFlag := "insecure"
```

Comment:
Should this var be renamed? A test for plain-http?

Reply:
I was meant to test the https->http auto handling via `--insecure`. This PR is pending for the discussion in https://github.com/oras-project/oras/issues/914, feel free to add your comment there.

Reply:
Right, but the var name is `plainHttpFlag` and the value is `insecure`